src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -279,6 +279,59 @@ def get_adjacent_releases_to_current_release(
             "prev_release_version": next_release_version,
         }
 
+    @staticmethod
+    def __get_top_of_queryset_release_version_based_on_order_by(org, filter_params, order_by):
+        """
+        Helper function that executes a query on Release table orders that query based on `order_by`
+        input provided
+        Inputs:-
+            * org: Organization object
+            * filter_params:
+            * order_by: Contains columns that are used for ordering to sort based on date
+        Returns:-
+            Release version of the top element of the queryset returned through ordering the Release
+            table by the order_by input
+        """
+        queryset = Release.objects.filter(
+            organization=org, projects__id__in=filter_params["project_id"]
+        )
+
+        queryset = add_environment_to_queryset(queryset, filter_params)
+
+        return queryset.order_by(*order_by).first().version
+
+    def get_first_and_last_releases(self, org, filter_params, sort):
+        """
+        Method that returns the first and last release based on `date_added`
+        Inputs:-
+            * org: organisation object
+            * filter_params
+            * sort: sort option i.e. date, sessions, users, crash_free_users and crash_free_sessions
+        Returns:-
+            A dictionary of two keys `first_release_version` and `last_release_version` representing
+            the first ever created release and the last ever created releases respectively
+        """
+        first_release_version = None
+        last_release_version = None
+
+        if sort == "date":
+            release_common_filters = {
+                "org": org,
+                "filter_params": filter_params,
+            }
```

Comment:
Would it make sense to just pass these values directly to the functions? I feel like this reduces readability

Reply:
Yeah I agree! changed